server/server.js

```diff
@@ -1400,7 +1409,7 @@ let needSetup = false;
                             await updateMonitorNotification(bean.id, notificationIDList);
 
                             // If monitor was active start it immediately, otherwise pause it
-                            if (monitorListData[i].active === 1) {
+                            if (monitorListData[i].isActive === 1) {
```

Comment:
`isActive` is a function which shouldn't exist here?

Reply:
Oh yeah, you're right. Fixing it directly.